i3-settings-qubes.spec.in

```diff
@@ -1,16 +1,17 @@
 Name:       i3-settings-qubes
-Version:    1.6
-Release:    2%{?dist}
+Version:    @VERSION@
+Release:    @REL@%{?dist}
 Summary:    Default i3 settings for Qubes
 
 Group:      User Interface/Desktops
 License:    GPLv2+
 URL:        http://www.qubes-os.org/
-Source0:    i3.config
-Source1:    i3.config.keycodes
-Source2:    qubes-i3-sensible-terminal
-Source3:    qubes-i3-xdg-autostart
-Source4:    qubes-i3status
+Source0:    %{name}-%{version}.tar.gz
+Source1:    i3.config
+Source2:    i3.config.keycodes
+Source3:    qubes-i3-sensible-terminal
+Source4:    qubes-i3-xdg-autostart
+Source5:    qubes-i3status
```

Comment:
Since those all will be included in the source tarball, no need to list them separately anymore.

Reply:
Yes indeed. I've but install part into Makefile for preparing Debian packaging